packages/components/data-table-manager/package.json

```diff
@@ -37,7 +37,7 @@
     "@emotion/core": "10.0.28",
     "@emotion/styled": "10.0.27",
     "debounce-promise": "^3.1.2",
-    "lodash.differencewith": "4.5.0",
```

Comment:
You have to fix the import to `lodash/differenceWith`

Reply:
Yeah, just saw that as the deployment failed 😅